src/app.ts

```diff
@@ -1,6 +1,13 @@
 import { startAppServer, startProxyServer } from './server/server'
+import dotEnv from './helpers/dotEnv'
 
-startAppServer(process.env.PORT || 3000, process.env.ADMIN)
-if (process.env.NODE_ENV === 'production') {
-  startProxyServer(process.env.HOME)
-}
+const { PORT, ADMIN_PASS, ENV, HOME } = dotEnv
+
+startAppServer(PORT, ADMIN_PASS)
+
+/**
+ * Proxy Server will create SSL Certificates on the server
+ * for your domains in production.
+ * Development mode do not have to run the proxy server.
+ * */
+if (ENV === 'production') startProxyServer(HOME)
```

Comment:
Personally, I avoid hard coded strings. Since you are refactoring the `env` file. could we just import `isProduction` function an run that? `if (isProduction())`